### PR TITLE
Support OWE (opportunistic wireless encryption) passwordless mode

### PR DESCRIPTION
**Pull request by vjr**
This OWE (opportunistic wireless encryption) passwordless mode is available with at least the 6 Ghz WiFi 6E band.

This PR goes with https://github.com/elementary/wingpanel-indicator-network/pull/254

### CODE REVIEW

File: src/Views/WifiPage.vala [hunk 1]
```diff
@@ -531,8 +532,24 @@ namespace Network {
                 wifi_dialog.run ();
                 wifi_dialog.destroy ();
             } else {
+                if (NM.@80211ApSecurityFlags.KEY_MGMT_OWE in row.ap.get_rsn_flags () ||
```
Comment (zeebok) [inline src/Views/WifiPage.vala:535]: This almost looks like it is replacing a different connection type? Is that right/a good idea?

Comment (vjr) [reply src/Views/WifiPage.vala:535]: From what I understand, it's not replacing, just adding the required properties (uuid and algorithm "owe") to the blank new connection before making the connection without the wifi dialog, if the owe mode is detected which doesn't have a password.

Similar to what is being done for WPA connections except there a wifi dialog is being shown to ask for the password.

Comment (zeebok) [reply src/Views/WifiPage.vala:535]: I see. I think this being in the else block after the is_secure check is making think this section was for open wifi APs. If that isn't the case *or* it is but this doesn't affect connecting to open networks, then I think this code seems alright. I sadly have no way to actually test though :(

**Review by zeebok — APPROVED** [review]

**Comment by zeebok** [comment]
The code looks fine but I cannot test this. I will leave the merging up to you.

**Comment by vjr** [comment]
Thank you @zeebok  !  I'll hold off until I, or anyone else, can actually test this.

**Comment by vjr** [comment]
@elementary/desktop-developers  I've minor update the PR to indicate to the user the OWE mode is "Passwordless encrypted" instead of "OWE unsecured" which is more appropriate and user-friendly IMHO.

**Comment by vjr** [comment]
Apologies for the re-review request but I've tested across older 2.4 ghz and wifi 6 and 6E (password WPA3, WPA2 and passwordless OWE modes) and these are working well.

**Review by zeebok — APPROVED** [review]

**Pull request merged**